app/services/irs_attempts_api/redis_client.rb

```diff
@@ -17,11 +17,15 @@ def write_event(event_key:, jwe:, timestamp:)
       end
     end
 
-    def read_events(timestamp:)
+    def read_events(timestamp:, batch_size: 5000)
```

Comment:
This is never passed in, but I'm leaving it to allow tinkering in `rails c`, unless rubocop yells.